### PR TITLE
KM3NeT pattern recognition optimization GSOC proposal 

### PR DESCRIPTION
**Pull request by roelaaij**

### CODE REVIEW

File: _gsocorgs/2019/nikhef.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Nikhef; Dutch national institute for subatomic physics"
+author: "Roel Aaij"
+layout: default
+organization: Nikhef
+logo: logo_Nikhef.jpg
+description: |
+[Nikhef](http://www.nikhef.nl) is the Dutch national institute for
```
Comment (aflp91) [inline _gsocorgs/2019/nikhef.md]: There are missing spaces in front of the sentence that prevent YAML reading…

File: _gsocproposals/2019/proposal_KM3NeTbeaming.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+organization:
+  - Nikhef
+---
+
+## Description
+
+KM3NeT [1] will house two next-generation underwater neutrino
+experiments. Two detectors are being constructed at two deep sites in
+the Mediterranean Sea, one of the coast of Sicily and the other south
+of Toulouse in Southern France. Nikhef [2] is involved the design and
```
Comment (aflp91) [inline _gsocproposals/2019/proposal_KM3NeTbeaming.md]: Typos: 
- Toulouse => Toulon
- is involved the => is involved in the

File: _gsocproposals/2019/proposal_KM3NeTbeaming.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+## Expected results
+A clear picture of performance bottlenecks in the algorithm and
+a implementation of a vectorized version.
+
+## Desirable Skills
+  * C++ knowledge
+  * Interest in pattern recognition algorithms
+  * Knowledge of writing C++ for vector instruction sets
+
+## Mentors
```
Comment (aflp91) [inline _gsocproposals/2019/proposal_KM3NeTbeaming.md]: Don't forget to add mentors in `mentors.md` file

File: _gsocorgs/2019/nikhef.md (hunk 1 above)
Comment (roelaaij) [reply _gsocorgs/2019/nikhef.md]: Fixed to be the same as in #474, I hope this avoids merge conflicts later

**Pull request merged**